features/diversity/disability_page.feature

```diff
@@ -0,0 +1,13 @@
+@wip @diversity @javascript
+
+Feature: Disability page
+
+Scenario: Successfully submit whether I have a disability
+  Given I am on the disability page
+  When I successfully submit whether I have a disability
+  Then I should be taken to the pregnancy and maternity page
+
+Scenario: Continue without providing an answer on my ethnic group
```

Comment:
I think this is a copy and paste error that I made, it should be -
Scenario: Continue without providing an answer to whether I have a disability, same goes for the steps, it should be able disability, not ethnic group.